samples/excel/30-range/conditional-formatting-basic.yaml

```diff
@@ -0,0 +1,390 @@
+                    The iconSetCF.criteria array is automatically prepopulated with
+                    criterion elements whose properties have been given default settings
+                    based on type passed to the add method above. In this snippet, iconSet.
+                    You cannot directly assign new values to the elements or their properties.
+                    You must first (1) assign the array to a variable, (2) modify the
+                    variable, and then (3) copy the variable back to the original array.
+                    Step #1 requires reading the iconSetCF.criteria, so you have to load it
+                    and sync before you can carry out the process.
+                */
+                iconSetCF.load("criteria");
+
+                await context.sync();
+
+                const criteria = iconSetCF.criteria;
+
+                /*
+                    With a "three*" icon set style, such as "threeFlags", the third
+                    element in the criteria array (criteria[2]) defines the "top" icon;
+                    e.g., a green flag. The second (criteria[1]) defines the "middle"
+                    icon, The first (criteria[0]) defines the "low" icon although it
+                    can often be left implicit as this method does below.
+                */
+
+                // Every cell value above 1000 gets a green flag
+                criteria[2].type = Excel.ConditionalFormatIconRuleType.number;
+                criteria[2].operator = Excel.ConditionalIconCriterionOperator.greaterThan;
+                criteria[2].formula = "=1000";
+
+                // Cell values between 701 and 1000 get a yellow flag
+                criteria[1].type = Excel.ConditionalFormatIconRuleType.number;
+                criteria[1].operator = Excel.ConditionalIconCriterionOperator.greaterThan;
+                criteria[1].formula = "=700";
+
+                /*
+                    Unless you are doing something special with the "low" icon criterion
+                    (criteria[0]) , you  do not need to explicitly set it's properties. Any cells
+                    that the middle and top criteria do not apply to will get the "low" icon. In
+                    this snppet everything below 700 gets a red flag.
+                */
+                iconSetCF.criteria = criteria;
+
+                await context.sync();
+            });
+        }
+
+        async function applyTextFormat() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const range = sheet.getRange("B16:D18");
+                const conditionalFormat = range.conditionalFormats
+                    .add(Excel.ConditionalFormatType.containsText);
+                conditionalFormat.textComparison.format.font.color = "red";
+                conditionalFormat.textComparison.rule = {operator: Excel.ConditionalTextOperator.contains, text: "Delayed"};
```

Comment:
I would split this over 2 lines, and also have space before the closing brace.  JS style is

    a = { b: 5 }